src/features/device-state/routes/state-get-v3.ts

```diff
@@ -0,0 +1,404 @@
+import type { RequestHandler } from 'express';
+import type { Request } from 'express';
+
+import * as _ from 'lodash';
+import {
+	captureException,
+	handleHttpErrors,
+} from '../../../infra/error-handling';
+import {
+	ConfigurationVarsToLabels,
+	filterDeviceConfig,
+	formatImageLocation,
+	getReleaseForDevice,
+	readTransaction,
+	rejectUiConfig,
+	serviceInstallFromImage,
+	setDefaultConfigVariables,
+	varListInsert,
+} from '../state-get-utils';
+import { sbvrUtils, errors } from '@balena/pinejs';
+import { events } from '..';
+import { Expand } from 'pinejs-client-core';
+
+const { UnauthorizedError } = errors;
+const { api } = sbvrUtils;
+
+type LocalStateApp = StateV3[string]['apps'][string];
+type ServiceComposition = AnyObject;
+export type StateV3 = {
+	[uuid: string]: {
+		name: string;
+		is_managed_by__device?: string;
```

Comment:
This field has a different name in the GET and PATCH PRs. On the GET PR it is named `parent_device` (which is what the supervisor is expecting). Is this PR just missing a rebase or you intend to change the name?

https://github.com/balena-io/open-balena-api/pull/687/files#diff-b645151f12e0231029c88685140b26ef52329c83bcabe726c1eced933329df52R32

Reply:
Yeah this is intended to be `is_managed_by__device` to match the model (as we do for other names) - I've moved the state-get change to the right commit/PR now